inbox/events/google.py

```diff
@@ -536,8 +537,8 @@ def parse_event_response(event: Dict[str, Any], read_only_calendar: bool) -> Eve
     creator = event.get("creator")
 
     if creator:
-        owner = "{} <{}>".format(
```

Comment:
This would not handle properly names containing characters like `<` i.e. pair `"Example <"`, `"test@example.com"` should be encoded as `'"Example <" <test@example.com>'` and not `"Example < <test@example.com>"`